help/blocks/toc/toc.css

```diff
@@ -0,0 +1,103 @@
+html {
+  scroll-behavior: smooth;
+}
+
+.section.article-long .article-main-wrapper {
+  display: flex;
+}
+
+.toc.block {
+  display: none;
+}
+
+.toc-content-wrapper {
```

Comment:
this could be preferably moved to `styles.css`

Reply:
Why would you move to the global styles? That's specific to the left column containing the ToC

Reply:
got it now! moved the variables to the corresponding section in styles.css